src/ToJSON/parsing/parsing.ts

```diff
@@ -58,7 +81,7 @@ export function ParseText(text?: string, parsingOptions?: string, invokeProgress
       invokeProgressFunction(lines.length, index);
     }
 
-    ProcessNewLine(lastLevel, lineNumber, line, nextLine);
+    ProcessNewLine(lastLevel, lineNumber, line, nextLine, replaceIdentifiersWithUUIDs);
```

Comment:
like in the other PR, dont add parameters... use a "global" options object instead